intro-OpenStack/ch_ConfigureInstances.tex

```diff
@@ -0,0 +1,154 @@
+\chapter{Configure access and security for instances}
+Openstack security groups are sets of IP filter rules that define
+networking access and are applied to all instances within a project.
+In the VSC cloud, each project contains a default security group,
+which allows you to ping instances and connect using SSH.  You can add
+rules to the default security group or add new security groups with
+rules.
+
+Key pairs are SSH credentials that can be automatically injected into
+an instance when it is launched. To use key pair injection, the image
+that the instance is based on must contain the \textbf{cloud-init}
+package. Each project should have at least one key pair. For more
+information, see the section \emph{Add a key pair} on page
+\pageref{add-a-key-pair}.
+
+If you have generated a key pair with an external tool, you can import
+it into \gls{OpenStack}. The key pair can be used for multiple
+instances that belong to a project. For more information, see the
+section {\emph{Import a key pair}} on page
+\pageref{import-a-key-pair}.  For general instructions on SSH keys, we
+refer to chapter 2 of
+\href{https://www.vscentrum.be/support/tut-book/vsc-tutorials}{the VSC
```

Comment:
maybe we should include a comment about the ssh keys from the account portal are injected automatically into openstack, so the users do not need to generate a new personal ssh key .